### PR TITLE
Chore: Favicon change

### PR DESCRIPTION
**Pull request by KarlGusta**
Changes made:
- Changed the favicon to the new logo favicon.

### CODE REVIEW

**Comment by PatelN123** [comment]
Hey @KarlGusta,
Thanks for this. I think the Sivu light bulb icon would look best here instead of using the whole logo. Do you agree?

I also think it would be better to load the logo from the website rather than GitHub storage.

File: pages/_app.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ const App = ({ Component, pageProps }) => {
       <Head>
         <meta content='IE=edge' httpEquiv='X-UA-Compatible' />
         <meta content='width=device-width, initial-scale=1' name='viewport' />
-        <link rel='icon' type='image/png' sizes='96x96' href='/favicon.png' />
+        <link rel='icon' type='image/png' sizes='96x96' href='https://user-images.githubusercontent.com/33565767/173816123-99707c2a-0bf0-46a0-a1c0-c4c4780beeb4.png' />
```
Comment (PatelN123) [inline pages/_app.tsx:20]: Can this be changed to a local path?
Also can we use just the lightbulb as the favicon here please.

**Review by PatelN123 — CHANGES_REQUESTED** [review]

File: pages/_app.tsx (hunk 1 above)
Comment (KarlGusta) [reply pages/_app.tsx:20]: Yes we can Patel. Thanks for your constructive feedback. I'll make the changes.

**Comment by PatelN123** [comment]
Sure, this works. It does look squished, but we can work on that later.

**Pull request merged**